.github/workflows/macos.yml

```diff
@@ -0,0 +1,69 @@
+name: macos
+on: [push, pull_request]
+
+jobs:
+
+  test:
+    strategy:
+      matrix:
+        platform:
+          - macos-12
+        generator:
+          - Ninja
+        configuration:
+          - Release
+    runs-on: ${{ matrix.platform }}
+    env:
+      # The `build` action requires these variables.
+      build_dir: .build
+      NUM_PROCESSORS: 2
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: install Ninja
+        if: matrix.generator == 'Ninja'
+        run: brew install ninja
+      - name: choose Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.9
+      - name: learn Python cache directory
+        id: pip-cache
+        run: |
+          sudo pip install --upgrade pip
+          echo "::set-output name=dir::$(pip cache dir)"
+      - name: restore Python cache directory
+        uses: actions/cache@v2
+        with:
+            path: ${{ steps.pip-cache.outputs.dir }}
+            key: ${{ runner.os }}-${{ hashFiles('.github/workflows/nix.yml') }}
+      - name: install Conan
+        run: pip install wheel 'conan<2'
+      - name: check environment
+        run: |
+          echo ${PATH} | tr ':' '\n'
+          python --version
```

Comment:
Throw this info on the workflow summary?

```
PYTHON_VERSION=$(python --version) 
CONAN_VERSION=$(conan --version)
CMAKE_VERSION=$(cmake --version)

echo "Python | Conan | CMake" >> $GITHUB_STEP_SUMMARY
echo ":-:|:-:|:-:" >> $GITHUB_STEP_SUMMARY
echo "${PYTHON_VERSION} | ${CONAN_VERSION} | ${CMAKE_VERSION}" >> $GITHUB_STEP_SUMMARY
```


`env >> $GITHUB_STEP_SUMMARY` :question:

Reply:
These can be multiline outputs (CMake is), and I don't feel like working around that. If it becomes something we check often and want to make easier to see, then I'll reconsider. Until then, I think what we have now is fine.